src/index.ts

```diff
@@ -187,18 +187,42 @@ export function declutter(arg: any): any {
   }
   let ret = { ...arg }
   // Marqeta can send back several empty fields on all responses... clean it
-  if (isEmpty(ret.nextUrl)) {
+  if (isEmpty(ret.metadata)) {
     delete ret.nextUrl
   }
-  if (isEmpty(ret.previousUrl)) {
-    delete ret.previousUrl
-  }
   if (ret.status === 200) {
     delete ret.status
   }
   return ret
 }
```

Comment:
I would **_strongly_** suggest that you not use `declutter()` unless every single response from Marqeta has fields that you don't want the caller to see. If there are, then this is a great way to do it... but `metadata` that might be something useful... Just a thought...

Oh... and if you are not getting back a `status` value, then this is never getting called:

```typescript
   if (ret.status === 200) {
     delete ret.status
   }
```

Right?